docs/development/compatibility-policy.rst

```diff
@@ -298,99 +303,105 @@ When an application wants to be upgraded to a new version of Twisted, it can do
-            "twisted.mail.protocols", "SSLContextFactory")
 
+    deprecatedModuleAttribute(
+        Version("Twisted", "NEXT", 0, 0),
+        "Use twisted.internet.ssl.DefaultOpenSSLContextFactory instead.",
+        __name__,
+        SSLContextFactory.__name__,
+    )
 
-Functions and methods
-^^^^^^^^^^^^^^^^^^^^^
+Pass ``Version("Twisted", "NEXT", 0, 0)`` `incremental placeholder <https://github.com/twisted/incremental#updating>`_ to the to indicate the upcoming release.
+In strings, ``Twisted NEXT`` works the same way.
 
-To deprecate a function or a method, add a call to warnings.warn to the beginning of the implementation of that method.
-The warning should be of type ``DeprecationWarning`` and the stack level should be set so that the warning refers to the code which is invoking the deprecated function or method.
-The deprecation message must include the name of the function which is deprecated, the version of Twisted in which it was first deprecated, and a suggestion for a replacement.
-If the API provides functionality which it is determined is beyond the scope of Twisted or it has no replacement, then it may be deprecated without a replacement.
+Functions and Methods
+^^^^^^^^^^^^^^^^^^^^^
 
-There is also a :py:func:`deprecated <twisted.python.deprecate.deprecated>` decorator which works for new-style classes.
+Use the :py:func:`deprecated <twisted.python.deprecate.deprecated>` decorator to deprecate methods.
 
 For example:
 
 .. code-block:: python
 
-    import warnings
-
+    from incremental import Version
     from twisted.python.deprecate import deprecated
-    from twisted.python.versions import Version
 
 
-    @deprecated(Version("Twisted", 1, 2, 0), "twisted.baz")
+    @deprecated(Version("Twisted", "NEXT", 0, 0), "twisted.baz")
     def some_function(bar):
         """
-        Function deprecated using a decorator.
+        Function deprecated using a decorator, replaced by twisted.baz.
         """
         return bar * 3
 
 
-
-    @deprecated(Version("Twisted", 1, 2, 0))
+    @deprecated(Version("Twisted", "NEXT", 0, 0))
     def some_function(bar):
         """
-        Function deprecated using a decorator and which has no replacement.
+        Function deprecated using a decorator which has no replacement.
         """
         return bar * 3
 
+If you can't use the decorator add a call to :py:func:`warnings.warn()` at the beginning of the implementation.
+The warning should be of type :py:exc:`DeprecationWarning` and the stack level should be set so that the warning refers to the code which is invoking the deprecated function or method.
+The deprecation message must include the name of the function which is deprecated, the version of Twisted in which it was first deprecated, and a suggestion for a replacement.
+If the API provides functionality which it is determined is beyond the scope of Twisted or it has no replacement, then it may be deprecated without a replacement.
```

Comment:
Maybe have this as a separate paragraph.

This looks like an important info. Maybe move it at the top of the document?

```suggestion

If the API provides functionality which it is determined is beyond the scope of Twisted or it has no replacement, then it may be deprecated without a replacement.
```